Packs/CofenseTriage/Integrations/CofenseTriage/CofenseTriage.py

```diff
@@ -51,145 +25,248 @@
     'category_id',
     'match_priority',
     'tags',
-    'email_attachments'
+    'email_attachments',
 ]
 
 
-# HELPER FUNCTIONS #
-def snake_to_camel_keys(snake_list: List[Dict]) -> List[Dict]:
-    def snake_to_camel(snake_str) -> str:
-        if snake_str == 'id':
-            return 'ID'
-        components = snake_str.split('_')
-        return ''.join(x.title() for x in components)
-
-    return [{snake_to_camel(k): v for k, v in snake_d.items()} for snake_d in snake_list]
-
-
-def split_snake(string: str) -> str:
-    return string.replace("_", " ").title()
+class TriageInstance:
+    def __init__(self, *, host, token, user, disable_tls_verification=False):
+        self.host = host
+        self.token = token
+        self.user = user
+        self.disable_tls_verification = disable_tls_verification
 
+    def request(self, endpoint, params=None, body=None, raw_response=False):
+        """
+        Make a request to the configured Triage instance and return the result.
+        """
+        # TODO automatic rate-limiting
```

Comment:
still missing?

Reply:
Scoped for a future release (if ever), to deliver incremental value to the customer sooner.